api/db/schema.prisma

```diff
@@ -34,17 +34,19 @@ model Image {
 }
 
 model ImageSurvey {
-  id               Int     @id @default(autoincrement())
+  id               Int      @id @default(autoincrement())
   privateRank      Int
   publicElem       String
   privateElem      String
   satisfactionRank Int?
   satisfactionElem String?
-  user             User?   @relation(fields: [userId], references: [id])
+  user             User?    @relation(fields: [userId], references: [id])
   userId           Int?
-  image            Image?  @relation(fields: [imageId], references: [id])
+  image            Image?   @relation(fields: [imageId], references: [id])
   imageId          Int?
   hasInterface     Boolean
+  createdAt        DateTime @default(now())
+  submittedAt      DateTime?
```

Comment:
You can use the `@updatedAt` directive to let prisma automatically update the field when the row is updated. it removes the hastle of having to set it manually. For the user it is different I guess, since the `submittedAt` is supposed to be set explicitly by a mutation and not updated for each SQL Update statement.

Reply:
Okay I will add it in Prisma and remove the manual setting